invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Breadcrumb } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export default class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
+    this.elements = props.elements;
+  }
+
+  _renderBreadcrumbElements = () => {
+    const lastElemIdx = this.elements.length - 1;
+    return this.elements.map((element, i) => (
```

Comment:
You can drop `lastElemIdx` and instead do:
```
return this.elements.map((element, i, arr) => {
    const icon = (i < arr.length - 1) ? 'chevron' : 'arrow';
    return (
        ...
        <Breadcrumb.Divider icon="right {icon}" />
        ...
    )}
);
```
`map` can have the input array as arg: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map